docs/SUMMARY.md

```diff
@@ -10,6 +10,7 @@
   * [Install using Kubectl](setup/install/install-devtron-using-kubectl.md)
   * [Installation Configurations](setup/install/installation-configuration.md)
   * [Override Configurations](setup/install/override-default-devtron-installation-configs.md)
+* [Devtron Upgrade](setup/Devtron-upgrade/README.md)
```

Comment:
Always use lower case in filenames in documentation. Also change the file `0.2.x-0.3.x.md` to `devtron-upgrade-0.2.x-0.3.x.md`